pkg/util/azure/azure_tags.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package azure
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type azureInstanceMetadata struct {
+	VMID          string `json:"vmId"`
+	Zone          string `json:"zone"`
+	VMSize        string `json:"vmSize"`
+	ResourceGroup string `json:"resourceGroupName"`
+}
+
+// GetTags gets the tags from the GCE api
+func GetTags() ([]string, error) {
```

Comment:
You will need to check the `cloud_provider_metadata` field like here: https://github.com/DataDog/datadog-agent/blob/master/pkg/util/azure/azure.go#L37